tox.ini

```diff
@@ -257,9 +252,25 @@ commands =
     pytest {toxinidir}/certbot-ci/certbot_integration_tests \
         --acme-server={env:ACME_SERVER:pebble} \
         --cov=acme --cov=certbot --cov=certbot_nginx --cov-report= \
-        --cov-config={toxinidir}/certbot-ci/certbot_integration_tests/.coveragerc \
-        -W 'ignore:Unverified HTTPS request'
-    coverage report --include 'certbot/*' --show-missing --fail-under=64
+        --cov-config={toxinidir}/certbot-ci/certbot_integration_tests/.coveragerc
+    coverage report --include 'certbot/*' --show-missing --fail-under=67
     coverage report --include 'certbot-nginx/*' --show-missing --fail-under=74
-passenv =
-    DOCKER_*
+passenv = DOCKER_*
+
+[testenv:integration-certbot-oldest]
+commands =
+    {[base]pip_install} .
+    {[base]pip_install} certbot-ci
+    pytest {toxinidir}/certbot-ci/certbot_integration_tests \
+        --acme-server={env:ACME_SERVER:pebble} --campaign=certbot -p no:sugar
+passenv = DOCKER_*
+setenv = CERTBOT_OLDEST=1
```

Comment:
This may depend on your answers to the above questions, but why not use `{[testenv:py27-oldest]setenv}` here?

Reply:
In the context of separating unit oldest tests and integration oldest tests, I think it would add unnecessarily an indirection between two tests suite that are (beside the oldest requirements), are not about the same subject.

Reply:
Deciding when it makes sense to couple things is subjective and kind of tricky in my opinion. Regardless of the approach taken, I think we should also be setting `PYTEST_ADDOPTS` to make use of multiple processes.

I think setting `PYTHONHASHSEED` which is the only other value in `{[testenv:py27-oldest]setenv}` isn't a bad idea either for reproducibility (see https://tox.readthedocs.io/en/latest/example/basic.html#special-handling-of-pythonhashseed), but I personally don't really care one way or another here.

Reply:
Here I see the interest to inherit `setenv` configs from more generic tox env. I added it, and use it to also configure specifically `PYTEST_ADDOPTS` during integration tests.